### PR TITLE
some adjusts for debug convenience

### PR DESCRIPTION
**Pull request by bbklab**
方便产品调试时定位代码
  - CLI: swan  version
  - API: /version

```bash
kill -USR1 `swan pid`
cat /tmp/swan-stack-trace.log
```

### CODE REVIEW

File: src/utils/swanerror.go [hunk 1]
```diff
@@ -14,23 +12,22 @@ const (
 
 type SwanError struct {
 	Severity SwanErrorSeverity `json:"Severity"`
-	Err      error             `json:"Err"`
```
Comment (bbklab) [inline src/utils/swanerror.go]: type `error` can't be json marshal

File: src/utils/util.go [hunk 2]
```diff
@@ -1,53 +1,38 @@
...
 )
 
+// LoadNodeID load or create id in local id file
 func LoadNodeID(filePath string) (string, error) {
-	if !fileutil.Exist(filePath) {
-		return "", errors.New("ID file was not found")
-	}
+	filePath = filepath.Clean(filePath)
 
-	idFile, err := os.Open(filePath)
```
Comment (bbklab) [inline src/utils/util.go]: forget `idFile.Close()`

File: src/utils/util.go [hunk 3]
```diff
@@ -1,53 +1,38 @@
...
 	}
-
-	logrus.Infof("starting swan node, ID file was found started with ID: %s", string(nodeID))
-
-	return string(nodeID), nil
-}
-
-func CreateNodeID(filePath string) (string, error) {
-	nodeID := uuid.NewV4().String()
-	idFile, err := os.OpenFile(filePath, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0644)
```
Comment (bbklab) [inline src/utils/util.go]: forget `idFile.Close()`

**Comment by cmingxu** [comment]
👍  LGTM

**Pull request merged**